### PR TITLE
Fix test workflows

### PR DESCRIPTION
**Pull request by lpusok**
### Checklist

- [x] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [x] `step.yml` and `README.md` is updated with the changes (if needed)

### Version

Requires new step ID, version 1.0.0 [version update](https://semver.org/)

### Context

Moved existing code from https://github.com/bitrise-steplib/bitrise-step-activate-bazel-cache. Will be released under a new Step ID in the steplib.
This Github repo is now managed from Terraform (https://github.com/bitrise-io/bitrise-github-steps/pull/139).
Added e2e test (the livegrep sample), that runs only on Ubuntu 20 at the moment, will have to fix this in a later PR.
The Step now uses the build cache CLI.

Removed the now unused BITRISE_CACHE_ENDPOINT output of the step.
Inputs are not yet added.

Example e2e run of Step: https://app.bitrise.io/build/94263faf-75dd-42d5-8c13-5f696b957ec1

### CODE REVIEW

**Review by viktorbenei — DISMISSED** [review]
lgtm 👍

File: bitrise.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - workflow: e2e
+
+  generate_readme:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-readme-generator.git: { }
+
+  run_step:
```
Comment (zsolt-marta-bitrise) [inline bitrise.yml:22]: What is this workflow's purpose? It doesn't seem to do much

File: step.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# 'read' has to be before 'set -e'
+read -r -d '' UNAVAILABLE_MESSAGE << EOF_MSG
+Bitrise Build Cache is not activated in this build.
+
+You have added the **Activate Bitrise Build Cache for Bazel** add-on step to your workflow.
+
+However, you don't have an activate Bitrise Build Cache Trial or Subscription for the current workspace yet.
```
Comment (zsolt-marta-bitrise) [inline step.sh:9]:
```suggestion
However, you don't have an active Bitrise Build Cache Trial or Subscription for the current workspace yet.
```

File: step.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# 'read' has to be before 'set -e'
+read -r -d '' UNAVAILABLE_MESSAGE << EOF_MSG
+Bitrise Build Cache is not activated in this build.
+
+You have added the **Activate Bitrise Build Cache for Bazel** add-on step to your workflow.
```
Comment (zsolt-marta-bitrise) [inline step.sh:7]: Why is it an add-on?

File: step.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+However, you don't have an activate Bitrise Build Cache Trial or Subscription for the current workspace yet.
+
+You can activate a Trial at [app.bitrise.io/build-cache](https://app.bitrise.io/build-cache),
+or contact us at [support@bitrise.io](mailto:support@bitrise.io) to activate it.
+EOF_MSG
+
+set -eo pipefail
+
+echo "Checking whether Bitrise Build Cache is activated for this workspace ..."
+if [ "$BITRISEIO_BUILD_CACHE_ENABLED" != "true" ]; then
```
Comment (zsolt-marta-bitrise) [inline step.sh:18]: I am not familiar with this flow, how is this variable being populated?

File: step.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+title: |-
```
Comment (zsolt-marta-bitrise) [inline step.yml:1]: Do we / can we / should we add a `maintainer: bitrise` too?

File: README.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+<details>
+<summary>Description</summary>
+
+This Step activates Bitrise's remote build cache add-on for subsequent Bazel executions in the workflow.
+
+After this Step executes, Bazel builds will automatically read from the remote cache and push new entries if it's enabled.
+
+</details>
+
+## 🧩 Get started
```
Comment (zsolt-marta-bitrise) [inline README.md:17]: Do we want to drop a line somewhere about the CLI? @viktorbenei ?

File: step.sh (hunk 4 above)
Comment (lpusok) [reply step.sh:18]: It is set here: https://github.com/bitrise-io/bitrise-website/blob/b2c588938831a7a35f626b80a72e5e119b2bde76/components/ci/app/helpers/workflow_helper.rb#L244

File: bitrise.yml (hunk 1 above)
Comment (lpusok) [reply bitrise.yml:22]: By convention there is a sample workflow for local testing in the steps (called `sample`), however this is not a very good one, so removed.

File: step.sh (hunk 3 above)
Comment (lpusok) [reply step.sh]: Not sure, we have the same one in the gradle step https://github.com/bitrise-steplib/bitrise-step-activate-gradle-remote-cache/blob/1f568f218c135875d221a9eec58dcac89c7de994/step.sh#L7.
Fixed it tough.

Comment (zsolt-marta-bitrise) [reply step.sh]: I'll add this to our unrelated ticket about the remote cache to update it there too with the same release

File: step.sh (hunk 4 above)
Comment (zsolt-marta-bitrise) [reply step.sh:18]: Thanks!

**Review by zsolt-marta-bitrise — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (lpusok) [reply README.md:17]: can do in a next changeset

**Pull request merged**